simulation/traffic_simulator/src/entity/misc_object_entity.cpp

```diff
@@ -53,6 +53,13 @@ void MiscObjectEntity::setBehaviorParameter(const traffic_simulator_msgs::msg::B
   THROW_SEMANTIC_ERROR("setBehaviorParameter function does not support in MiscObjectEntity.");
 }
 
+auto MiscObjectEntity::getDefaultDynamicConstraints() const
+  -> const traffic_simulator_msgs::msg::DynamicConstraints &
+{
+  THROW_SEMANTIC_ERROR(
+    "getDefaultDynamicConstraints function does not support in MiscObjectEntity.");
+}
```

Comment:
Why does it raise an error instead of returning a default-constructed DynamicConstraints object as the function name suggests?

Reply:
The default value is determined by the type of Entity, such as vehicle or pedestrian.
MiscObjects are not intended to move, so it is not possible to get a value for DynamicConstraints in the first place.
Therefore, an error is thorwn.

Reply:
If the MiscObject entity doesn't move, why not express this by returning DynamicConstraints with all zero values?

Reply:
OK, I changed to return DynamicConstraints with all zero values.

Reply:
You forgot to set max_speed.